integration-cli/docker_api_resize_test.go

```diff
@@ -10,7 +10,7 @@ import (
 
 func (s *DockerSuite) TestResizeApiResponse(c *check.C) {
 	testRequires(c, DaemonIsLinux)
```

Comment:
@vdemeester Weird. Not sure why that didn't hit the PR. Updated, waiting for confirmation from the windowsTP4 context run.